### ISSUE TITLE
[BUG] Remove unnecessary _get_properties code

### ISSUE THREAD

**Issue by mthibaut** [salt-extensions/saltext-proxmox#8]
**Description**
Remove unnecessary proxmox.py code regarding static_props in _get_properties.

**Additional context**
In _get_properties the following is done:

- _get_properties is passed an optional extra list of properties (static_props)
- It then checks the API for properties that can be set, adding ones that are and ignoring any on the extra list that aren't 
- The remaining params are returned

In other words the extra list is completely ignored in a roundabout way. Just remove it.

This issue is mentioned in https://github.com/saltstack/salt/issues/56633
